recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: 0
-  skip: true          # [win and not (py34 or py35)]
+  skip: true          # [win and py27]
```

Comment:
This was added because we only wanted to build with `py34 or py35` and not with other versions like `py36` now.